ArduCopter/radio.cpp

```diff
@@ -129,6 +135,17 @@ void Copter::read_radio()
         return;
     }
 
+    if (timeout == FS_RADIO_TIMEOUT_MS) {
+        // RC Radio
+        if (!is_change_flight_mode) {
+            set_mode(Mode::Number::BRAKE, ModeReason::RADIO_FAILSAFE);
```

Comment:
set_mode can fail so probably need to handle the failure or add a comment saying that it is ok to fail.